client/web/compose/src/components/PageBlocks/Navigation/Configurator.vue

```diff
@@ -0,0 +1,300 @@
+    <b-tab :title="$t('navigation.label')">
+      <div class="mb-3">
+        <h5 class="text-primary">
+          {{ $t("navigation.displayOptions") }}
+        </h5>
+
+        <b-row class="justify-content-between">
+          <b-col
+            cols="6"
+            sm="4"
+            class="mb-2 mb-sm-0"
+          >
+            <b-form-group
+              horizontal
+              :label="$t('navigation.appearance')"
+            >
+              <b-form-radio-group
+                v-model="options.display.appearance"
+                buttons
+                button-variant="outline-secondary"
+                size="sm"
+                name="appearance"
+                :options="appearanceOptions"
+              />
+            </b-form-group>
+          </b-col>
+
+          <b-col
+            cols="6"
+            sm="4"
+            class="mb-2 mb-sm-0"
+          >
+            <b-form-group
+              horizontal
+              :label="$t('navigation.alignment')"
+            >
+              <b-form-radio-group
+                v-model="options.display.alignment"
+                buttons
+                button-variant="outline-secondary"
+                size="sm"
+                name="alignment"
+                :options="alignmentOptions"
+              />
+            </b-form-group>
+          </b-col>
+
+          <b-col
+            cols="6"
+            sm="4"
+            class="mb-2 mb-sm-0"
+          >
+            <b-form-group
+              horizontal
+              :label="$t('navigation.fillJustify')"
+            >
+              <b-form-radio-group
+                v-model="options.display.fillJustify"
+                buttons
+                button-variant="outline-secondary"
+                size="sm"
+                name="fill-justify"
+                :options="fillJustifyOptions"
+              />
+            </b-form-group>
+          </b-col>
+        </b-row>
+      </div>
+      <div class="mb-3 mt-2">
+        <div class="d-flex align-items-center mb-4">
+          <h5 class="text-primary mb-0">
+            {{ $t("navigation.listItems") }}
+          </h5>
+
+          <b-button
+            variant="link"
+            class="d-flex align-items-center text-decoration-none"
+            @click="handleAddButton"
+          >
+            <font-awesome-icon
+              :icon="['fas', 'plus']"
+              size="sm"
+              class="mr-1"
+            />
+            {{ $t("navigation.add") }}
+          </b-button>
+        </div>
+
+        <div class="mt-3">
+          <draggable
+            v-model="block.options.lists"
+            group="sort"
+            handle=".grab"
+          >
+            <div
+              v-for="(column, index) in block.options.lists"
+              :key="`column-${index}`"
+            >
+              <hr v-if="index" />
+
+              <table
+                class="
+                  table table-sm table-borderless table-responsive-lg
```

Comment:
if you use b-table-simple i think you can replace some of those classes with props, do check it out